docs_src/getting-started/quick-start/index.md

```diff
@@ -244,7 +244,51 @@ EdgeX Foundry is an operating system (OS)-agnostic and hardware (HW)-agnostic Io
 !!! Info
     EdgeX is agnostic with regards to hardware (x86 and ARM), but only release artifacts for x86 and ARM 64 systems.  EdgeX has been successfuly run on ARM 32 platforms but has required users to build their own executables from source.  EdgeX does not officially support ARM 32.
 
-## REFERENCE - Default Service Ports
+## REFERENCE EdgeX Container Names
```

Comment:
Love this section!

Reply:
What do you think of creating a header of Reference and nesting these sections underneath it?